Bottom: 68f8c9e4bacadf9557668af3d789ced377646cde
Top:    2db3ecfe88c64a5860045bcd32214ce536ba35ab
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 00:10:46 +0100

Refresh of crosshair-c-use-a-dummy-via-st

---

diff --git a/src/crosshair.c b/src/crosshair.c
index f1e3ce7..3d352b6 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -94,7 +94,7 @@ thindraw_moved_pv (PinType *pv, int x, int y)
   moved_pv.X += x;
   moved_pv.Y += y;
 
-  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, true, false);
+  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, false, false);
 }
 
 /* ---------------------------------------------------------------------------
